Dockerfile

```diff
@@ -13,15 +11,16 @@ ENV HOME /home/lounge
 ENV LOUNGE_HOME ${HOME}/data
 ENV LOUNGE_SRC ${HOME}/src
 
-RUN mkdir -p ${LOUNGE_SRC}
-WORKDIR ${LOUNGE_SRC}
-
-RUN curl -L https://github.com/thelounge/lounge/archive/v${LOUNGE_VERSION}.tar.gz | tar --strip-components 1 -xzf -
-
-RUN chown -R lounge:lounge ${LOUNGE_SRC}
+# Create a non-root user for lounge to run in.
+RUN useradd --create-home --uid 1000 lounge
```

Comment:
Meh, I'm usually in favor of letting the OS decide what uid to give as it might conflict with other software on the machine, but since we are talking about lightweight containers that are well compartmentalized, maybe the practices are different, I'm not familiar.

I'll be OK with this if other @thelounge/maintainers do not object.